### PR TITLE
fix: Fix the correcrt link to the docs

### PR DESCRIPTION
**Pull request by ykarlinski-newrelic**
# Summary
We had a 404 with the docs. PR is the fix

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/mlops/sagemaker/config.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ keywords:
 
 documentation:
   - name: Sagemaker Docs
-    url: https://docs.newrelic.com/docs/alerts-applied-intelligence/mlops/integrations/aws-sagemaker-mlops-integration/
+    url: https://docs.newrelic.com/docs/mlops/integrations/aws-sagemaker-mlops-integration/
     description: Documentation on the Amazon SageMaker integration
```
Comment (jpvajda) [inline quickstarts/mlops/sagemaker/config.yml:37]:
```suggestion
    description: | 
        Documentation on the Amazon SageMaker integration
```

File: quickstarts/mlops/sagemaker/config.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@ keywords:
 
 documentation:
   - name: Sagemaker Docs
-    url: https://docs.newrelic.com/docs/alerts-applied-intelligence/mlops/integrations/aws-sagemaker-mlops-integration/
+    url: https://docs.newrelic.com/docs/mlops/integrations/aws-sagemaker-mlops-integration/
```
Comment (jpvajda) [inline quickstarts/mlops/sagemaker/config.yml:36]:
```suggestion
    url:  >- 
       https://docs.newrelic.com/docs/mlops/integrations/aws-sagemaker-mlops-integration/
```

**Review by jpvajda — CHANGES_REQUESTED** [review]

**Comment by jpvajda** [comment]
I made some suggestions to the description and URL format under documentation, please review the [contributor guide](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#documentation) for details on why I am suggestion this. Thanks for the contribution.

**Comment by ykarlinski-newrelic** [comment]
Commiting reviewer changes

**Review by jpvajda — APPROVED** [review]

**Pull request merged**